hw/dv/data/common_sim_cfg.hjson

```diff
@@ -23,12 +23,17 @@
   sw_root_dir:        "{proj_root}/sw"
 
   // pass and fail patterns
-  pass_patterns:    ["^TEST PASSED (UVM_)?CHECKS$"]
-  fail_patterns:    ["^UVM_ERROR\\s[^:].*$",
-                     "^UVM_FATAL\\s[^:].*$",
-                     "^Assert failed: ",
-                     "^\\s*Offending '.*'",
-                     "^TEST FAILED (UVM_)?CHECKS$"]
+  compile_pass_patterns: []
```

Comment:
```suggestion
  build_pass_patterns: []
```

Reply:
Reasoning: make it consistent with existing "build_modes", "build_opts", "run_modes", "run_opts" etc. 
"Sim" really refers to the whole simulation which includes build and run steps.

Reply:
make sense, updated.